Remotion/Web/Core/UI/Controls/TabbedMultiView.cs

```diff
@@ -268,11 +268,13 @@ private void OnTabViewInserted (TabView view)
       tab.Target = view.ID;
       _tabStrip.Tabs.Add(tab);
 
-      if (Views.Count == 2 && Views.IndexOf(_placeHolderTabView) > 0)
+      if (Views.Count == 2)
         Views.Remove(_placeHolderTabView);
-
-      if (Views.Count == 1)
+      else if (Views.Count == 1)
```

Comment:
We can combine those IFs because the OnTabViewRemoved handler takes are of the logic, correct? I think we should add assertions in the IF block that activeViewIndex == 0 and the first view is selected.